models/metafields/shopify__product_image_metafields.sql

```diff
@@ -0,0 +1,6 @@
+{{ config(enabled=var('shopify_using_all_metafields', False) or var('shopify_using_product_image_metafields', False)) }}
+
+{{ get_metafields( 
+    source_object = "stg_shopify__product_image", 
+    reference_value = 'image') 
```

Comment:
does this need to be `product_image`?

Reply:
I was stumped on this one... The data we have does not actually have any metafields for product_image. However, the product_variant data was shown as `variant`. Therefore, I assumed `image` would be the translation for product_image.

Reply:
Based on the [Shopify docs](https://shopify.dev/api/liquid/objects/image) we will keep this as just `image`